app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -384,19 +386,19 @@ public void onDeleteButtonClicked(){
         if(isDeleted) {
             dialog.getButton(AlertDialog.BUTTON_POSITIVE).setEnabled(false);
         }
+        }
         //Reviewer correct me if i have misunderstood something over here
         //But how does this  if (delete.getVisibility() == View.VISIBLE) {
         //            enableDeleteButton(true);   makes sense ?
+        else{
         AlertDialog.Builder alert = new AlertDialog.Builder(getActivity());
-        alert.setMessage("Why should this fileckathon-2018  be deleted?");
+        alert.setMessage("Why should "+ media.getDisplayTitle() +"  be deleted?");
```

Comment:
It would be great if you move this hard coded string into strings.xml file in another pull request @sherlockbeard

Reply:
@neslihanturan  i added the string value in #3015